BrickController2/BrickController2/BusinessLogic/PlayLogic.cs

```diff
@@ -235,6 +235,8 @@ private void SetPreviousOutput(ControllerAction controllerAction, float value)
                 }
             }
 
+            axisValue = axisValue * 100F / controllerAction.MaxInputPercent; 
```

Comment:
I would add an extra check if MaxInputPercent is not zero.

Reply:
Good point, now added.

Although I now realise a related issue which is that this field will default to 0, and so will break existing creations, as all inputs will be scaled to nothing.  

I'm not familiar with C# or the SQLite ORM, so I'm not sure how to set a default for the column.  I've manually implemented defaulting in d23e600 but I suspect that there's a cleaner way.